api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1801,6 +1805,15 @@ func (cluster *FoundationDBCluster) GetMaxConcurrentReplacements() int {
 	return pointer.IntDeref(cluster.Spec.AutomationOptions.MaxConcurrentReplacements, math.MaxInt64)
 }
 
+// UseManagementAPI returns the value of UseManagementAPI or false if unset.
+func (cluster *FoundationDBCluster) UseManagementAPI() bool {
+	if cluster.Spec.AutomationOptions.UseManagementAPI == nil {
+		return false
+	}
+
```

Comment:
```suggestion
```

Can be removed that is handled by `pointer.BoolDeref` and the whole reason we use it :)